docs/Userguide_sharing_data.rst

```diff
@@ -0,0 +1,101 @@
+Sharing Data with ACLs
+======================
+
+As an illustrative example, to allow ``$USER`` to share with ``$USER2`` in
+``rwx`` a hierarchy ``/network/scratch/$USER/X/Y/Z/...`` with ACLs:
+
+----
+
+| Grant **oneself** permissions to access any **future** files/folders created
+  by the other *(or oneself)*
+| (``-d`` renders this permission a "default" / inheritable one)
+
+.. code-block:: bash
+
+   setfacl -Rdm user:$USER:rwx  /network/scratch/$USER/X/Y/Z/
+
+----
+
+| Grant **another** permission to access any **future** files/folders created
+  by the other *(or oneself)*
+| (``-d`` renders this permission a "default" / inheritable one)
+
+.. code-block:: bash
+
+   setfacl -Rdm user:$USER2:rwx /network/scratch/$USER/X/Y/Z/
+
+----
+
+| Grant **another** permission to access any **existing** files/folders created
```

Comment:
I would put how to handle *existing files* _before_ *future files*

Reply:
See a comment above